media-tv/plex-media-server/plex-media-server-1.22.2.4282.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit systemd unpacker
+
+DESCRIPTION="A free media library that is intended for use with a plex client"
+HOMEPAGE="https://www.plex.tv/"
+
+MY_COMMIT="a97b03fad"
+MY_PV="${PV}-${MY_COMMIT}"
+
+MY_URI="https://downloads.plex.tv/plex-media-server-new"
+SRC_URI="
+	amd64? ( ${MY_URI}/${MY_PV}/debian/plexmediaserver_${MY_PV}_amd64.deb )
+	x86? ( ${MY_URI}/${MY_PV}/debian/plexmediaserver_${MY_PV}_i386.deb )"
+
+LICENSE="Plex"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+RESTRICT="mirror bindist strip"
+IUSE="system-openssl"
+
+BDEPEND="dev-util/patchelf"
+RDEPEND="
+	acct-group/plex
+	acct-user/plex
+	system-openssl? ( dev-libs/openssl )
+"
+
+QA_PREBUILT="*"
+QA_MULTILIB_PATHS=(
+    "usr/lib/plexmediaserver/lib/.*"
+    "usr/lib/plexmediaserver/Resources/Python/lib/python2.7/.*"
+    "usr/lib/plexmediaserver/Resources/Python/lib/python2.7/lib-dynload/_hashlib.so"
```

Comment:
These are whitespaces instead of tabs. Be sure to fix these (same thing at line 40)

Reply:
Fixed, thanks.